docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -1003,16 +1028,18 @@ thor: []
         <para>The costs incurred by a workunit for accessing logical files is
         also recorded in the workunit’s statistics and attributes. The
         read/write cost is recorded at the activity record and cumulated at
-        the graph, the subgraph and the workflow scope level. The total file
-        access cost for a workunit is recorded with the workunit and displayed
-        in the summary page.</para>
-
-        <para>The new cost field is shown in the Logical File summary page. It
-        is the combined cost of storing and accessing the data.</para>
-
-        <para>The cost information is currently only generated for Thor and
-        hThor jobs.</para>
-      </sect3>
+        the graph, the subgraph and the workflow scope level.
+        Notable improvement to the File Access Cost is files that are accessed from the page cache doesn’t incur file access cost.  
```

Comment:
this sentence is not needed:
Notable improvement to the File Access Cost is files that are accessed from the page cache doesn’t incur file access cost.